go/vt/vttablet/tabletserver/tabletserver.go

```diff
@@ -769,6 +769,14 @@ func (tsv *TabletServer) SetDemotePrimaryStalled() {
 	tsv.BroadcastHealth()
 }
 
+// ClearDemotePrimaryStalled clears the demote primary stalled field in the state manager.
```

Comment:
Unless you want calls to this to be very clear and intentional, there's no reason to not just change `Set` to take a boolean value and apply it. No need to add a new func.

Reply:
Nah, its the same for testing too, I'll make the change.

Reply:
Fixed